enterprise/server/remote_execution/containers/soci_store/soci_store.go

```diff
@@ -0,0 +1,55 @@
+package soci_store
```

Comment:
nit: the `containers` dir currently contains one package per supported isolation type. Seems like this pkg might fit better directly under remote_execution/?